actor/src/main/java/org/apache/pekko/japi/JAPI.java

```diff
@@ -15,8 +15,14 @@
 
 import scala.collection.Seq;
 
+@Deprecated
 public class JAPI {
 
+  /**
+   * Use <code>Util.immutableSeq(java.util.Arrays.asList(...))</code> instead. Deprecated since
```

Comment:
```suggestion
   * @deprecated Use <code>Util.immutableSeq(java.util.Arrays.asList(...))</code> instead. Deprecated since
```

Reply:
remove the `@Deprecated` annotation if the compiler won't let you use both

What is `Util` - can you fully reference it by including the package name?